native/core/src/execution/shuffle/row.rs

```diff
@@ -904,7 +904,7 @@ pub(crate) fn append_field(
                     append_map_element!(StringBuilder, Decimal128Builder, field);
                 }
                 (DataType::Utf8, DataType::Struct(_)) => {
-                    append_map_element!(StringBuilder, StructBuilder, field);
+                    //append_map_element!(StringBuilder, StructBuilder, field);
```

Comment:
Map arm branches takes too much of space in this file, proposing to move Map arm branches into a separate file